pypdf/_page.py

```diff
@@ -1793,19 +1816,25 @@ def _extract_text(
         char_scale = 1.0
         space_scale = 1.0
         _space_width: float = 500.0  # will be set correctly at first Tf
+        _actual_str_size: Dict[str, float] = {
+            "str_widths": 0.0, "space_width": 0.0, "str_height": 0.0}  # will be set string length calculation result
```

Comment:
This sounds odd at first sight. Did you mean this?

```suggestion
            "str_widths": 0.0, "space_width": 0.0, "str_height": 0.0}  # will be set to string length calculation result
```